UI/OptionsWnd.cpp

```diff
@@ -489,65 +489,66 @@ OptionsWnd::OptionsWnd():
     GG::Connect(show_font_texture_button->LeftClickedSignal, &ShowFontTextureWnd);
 
     CreateSectionHeader(current_page, 0, UserString("OPTIONS_FONT_SIZES"));
-    IntOption(current_page, 0, "UI.font-size",       UserString("OPTIONS_FONT_TEXT"));
```

Comment:
Completely unrelated cosmetic change.  Next time please make a separate PR for things like that.